src/api/world.py

```diff
@@ -76,9 +98,22 @@ def exit(self):
         shutil.rmtree(get_temp_dir(self._directory), ignore_errors=True)
 
     @property
-    def block_definitions(self) -> numpy.ndarray:
+    def block_definitions(self) -> BlockManager:
+        """
+        Allows access to the :class:`api.blocks.BlockManager` instance for this World
+        :return: The instance of the :class:`api.blocks.BlockManager`
+        """
         return self._wrapper.mapping_handler
 
+    def get_block_instance(self, blockstate: str) -> Block:
```

Comment:
This should also be changed to return a string (The code here stays the same, only the returned value changes)

Reply:
Why would returning a string be better here? Most use cases will convert that string to a Block object anyway, so why no keep it as-is?

Reply:
Right but each version will have its own "get_blockstate" function that will need to use the Block class. If we'll change something about the Block class, all of them will need to be updated as well. I prefer to only update the functions that actually use the Block object and not the classes that handle the data stuff.

Reply:
Would it better to have `get_blockstate()` in `WorldFormat` to return a string then in this method use that string to create the `Block` object and return it?

Reply:
Yeah, that can work. As long as the version-specific function that returns a string is not private in case someone prefers to use it.